<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test1Class</class>
 <widget class="QMainWindow" name="test1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="show_all_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>显示所有</string>
    </property>
   </widget>
   <widget class="QPushButton" name="query_button">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>260</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>610</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>记录ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="id_textEdit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>130</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="device_id_textEdit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>130</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="device_name_textEdit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>200</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>200</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>260</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_buttton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>410</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify_buttton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>500</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>show_all_button</sender>
   <signal>clicked()</signal>
   <receiver>test1Class</receiver>
   <slot>show_all_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_button</sender>
   <signal>clicked()</signal>
   <receiver>test1Class</receiver>
   <slot>exit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1210</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>query_button</sender>
   <signal>clicked()</signal>
   <receiver>test1Class</receiver>
   <slot>query_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>test1Class</receiver>
   <slot>add_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1102</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>1102</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_buttton</sender>
   <signal>clicked()</signal>
   <receiver>test1Class</receiver>
   <slot>delete_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>930</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>931</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modify_buttton</sender>
   <signal>clicked()</signal>
   <receiver>test1Class</receiver>
   <slot>modify_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1081</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>1070</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_all_clicked()</slot>
  <slot>exit_clicked()</slot>
  <slot>query_clicked()</slot>
  <slot>add_clicked()</slot>
  <slot>delete_clicked()</slot>
  <slot>modify_clicked()</slot>
 </slots>
</ui>
